optional_requirements.txt

```diff
@@ -1,6 +1,6 @@
-PySide; python_version != '2.7'
+PySide; python_version != '3.5' and python_version != '3.6'
 imread; python_version != '2.7'
-SimpleITK; python_version != '3.4' and python_version != '3.5'
+SimpleITK; python_version != '3.4' and python_version != '3.5' and python_version != '3.6'
```

Comment:
SimpleITK seems to have a support for 3.4: https://pypi.python.org/pypi/SimpleITK/0.9.1 .